src/components/chat/chat-window.tsx

```
@@ -0,0 +1,220 @@
+    },
+  });
+  
+  // Listen for new messages via Pusher
+  useEffect(() => {
+    if (!roomId) return;
+    
+    try {
+      const pusher = getPusherClient();
+      const channel = pusher.subscribe(`chat-room-${roomId}`);
+      
+      channel.bind("new-message", (data: { message: any }) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)